### ISSUE TITLE
RqSocket.getRemotePort() throws exception

### ISSUE THREAD

**Issue by EugenDueck** [yegor256/takes#1060]
```
org.takes.HttpException: [400] header "X-Takes-RemotePort" is mandatory, not found among [host, x-takes-localport, x-takes-remoteaddress, user-agent, accept, x-takes-localaddress]
        at org.takes.rq.RqHeaders$Smart.single(RqHeaders.java:226)
        at org.takes.rq.RqSocket.getRemotePort(RqSocket.java:93)
```

**Comment by yegor256** [comment]
@EugenDueck I believe, this is fixed in #1061 Thanks!

**Comment by 0crat** [comment]
Job `gh:yegor256/takes#1060` is not assigned, can't get performer
